.editorconfig

```diff
@@ -92,3 +92,6 @@ trim_trailing_whitespace = unset
 
 [pkgs/tools/misc/timidity/timidity.cfg]
 trim_trailing_whitespace = unset
+
+[pkgs/games/minetest/packages/contentdb/default.nix]
```

Comment:
The script that generates this file should be fixed so that it doesn't include any whitespace.

Reply:
Why? Other generated files get the exception, the trailing whitespace does no harm since the file is always regenerated in the same way, and removing it would make the generator script less readable.

Reply:
The others (only `eggs.nix` files and `hackage-packages.nix`) were pre-existing when we added this check and they won't be exemption when we start formatting `.nix` files. 

> make the generator script less readable.

Could just format the file which would remove the whitespace.

Reply:
> Could just format the file which would remove the whitespace

That's definitely cleaner, but I'd still like to know _why_  that is useful for a generated file.
In NixOS/rfcs#0101 it was decided not to format generated files, and there is still no consensus on the style/formatter to use, so why should we add complexity now, at the risk of having to change the formatter later?

Reply:
> I'd still like to know why that is useful for a generated file.

What benefit is there in excluding files? Isn't it easier if the entire tree is consistent? Currently there are only 5 nix files excluded because of whitespace, I can easily reduce that to just one (haskell package set). If haskell is fixed do you still think this file should to be excluded?

> it was decided not to format generated files

Where does it day that?

> so why should we add complexity now

Doesn't seem complicated.

> at the risk of having to change the formatter later

What does it matter if it needs to be changed?

Reply:
> What benefit is there in excluding files? Isn't it easier if the entire tree is consistent?

I think it isn't really easier for anyone, since nobody is going to manually touch that file.

But...

> If haskell is fixed do you still think this file should to be excluded?

Probably not

> Where does it day that?

Looks like I can't find it anymore... I only find conflicting info about that now

> Doesn't seem complicated.

Fair enough, and this is really a minor thing so I don't know why I was so stubborn...

Fixed. Now the extra whitespaces are filtered out.